# 1) Set map 'mapA', 'mapB', 'mapC'
# 2) Set 'cop', 'rob', or 'no' (include the single quotes)
# 3) Set goal_planner to 'pomdp' 'simple' 'stationary' 'rob_int' ... etc

map: mapA
cop_rob: # label as 'rob' or 'cop' (INCLUDE SINGLE QUOTES)
    Deckard: 'cop'
    Pris: 'no'
    Roy: 'no'
    Zhora: 'rob'
robots:
  Deckard:
    goal_planner: pomdp
  Pris:
    goal_planner: stationary
  Roy:
    goal_planner: simple
  Zhora:
    goal_planner: simple

